### PR TITLE
fixed double jump

### PR DESCRIPTION
**Pull request by python3-Trying-py**
fixed the double jump because someone updated the coyote time and completely broke it

### CODE REVIEW

File: Scripts/AirPlayer.gd [hunk 1]
```diff
@@ -6,7 +6,12 @@ func _ready():
...
 	default_coyote_time = INF
 	
-func djump():
-	move_cooldown = default_move_cooldown
-	if is_on_floor() == false:
-		jump()
+var hasdjump: bool = true
+
+func jump() -> void:
+	velocity.y = max(velocity.y, 0) + 15
```
Comment (urlordjames) [inline Scripts/AirPlayer.gd]: Please use the `default_jump_strength` variable (defined in Player) instead of 15.

**Review by urlordjames — CHANGES_REQUESTED** [review]
I merged my changes from main into your branch already.

File: Scripts/AirPlayer.gd (hunk 1 above)
Comment (python3-Trying-py) [reply Scripts/AirPlayer.gd]: `default_jump_strength` doesn't exist

Comment (urlordjames) [reply Scripts/AirPlayer.gd]: `git pull` my guy

Comment (urlordjames) [inline Scripts/AirPlayer.gd]:
```suggestion
	velocity.y = max(velocity.y, 0) + default_jump_strength
```

**Review by urlordjames — CHANGES_REQUESTED** [review]
I think you can just one-click accept this.  Never tried this feature before.

**Review by urlordjames — APPROVED** [review]
LGTM

**Pull request merged**